src/bentoml/_internal/frameworks/pytorch.py

```diff
@@ -195,15 +208,25 @@ def get_runnable(bento_model: Model):
     from .common.pytorch import PytorchModelRunnable
     from .common.pytorch import make_pytorch_runnable_method
 
-    partial_kwargs: t.Dict[str, t.Any] = bento_model.info.options.partial_kwargs  # type: ignore
+    opts = t.cast(ModelOptions, bento_model.info.options)
+    if get_pkg_version("torch") >= "2.0.0":
+        _load_model = partial(
+            load_model,
+            fullgraph=opts.fullgraph,
+            dynamic=opts.dynamic,
+            backend=opts.backend,
+            mode=opts.mode,
+            options=opts.options,
+            disable=opts.disable,
+        )
+    else:
+        _load_model = load_model
```

Comment:
I think we can just `model = load_model`, and then `torch.compile(model, **compile_kwargs)` if `enable_compile` is True.